maliput_ros_translation/package.xml

```diff
@@ -0,0 +1,35 @@
+<?xml version="1.0"?>
+<?xml-model href="http://download.ros.org/schema/package_format3.xsd" schematypens="http://www.w3.org/2001/XMLSchema"?>
+<package format="3">
+  <name>maliput_ros_translation</name>
+  <version>0.1.0</version>
```

Comment:
nit: 
```suggestion
  <version>0.0.1</version>
```
This way when we proceed to do a bloom-release we move minor version and we have a 0.1.0 release.